src/App.js

```diff
@@ -266,19 +297,38 @@ function App() {
     persistLocalStorage,
   ]);
 
+  useEffect(() => {
+    let wallet = initWallet();
+    let initializationAttempts = 0;
+
+    if (!wallet) {
+      const intervalID = setInterval(() => {
```

Comment:
Before, initialization might fail, and the page would break, because we tried to access `window.arweaveWallet` too early (which is a problem when using a development build).